src/repair/sources.rs

```diff
@@ -114,19 +121,33 @@ fn insert_entry<P: AsRef<Path>>(
     Ok(())
 }
 
-fn create_new_sources_list(release: &str) -> io::Result<()> {
-    fs::write(MAIN_SOURCES, format!(
-        "deb http://us.archive.ubuntu.com/ubuntu/ {0} restricted multiverse universe main\n\
-         deb-src http://us.archive.ubuntu.com/ubuntu/ {0} restricted multiverse universe main\n\
-         deb http://us.archive.ubuntu.com/ubuntu/ {0}-updates restricted multiverse universe main\n\
-         deb-src http://us.archive.ubuntu.com/ubuntu/ {0}-updates restricted multiverse universe main\n\
-         deb http://us.archive.ubuntu.com/ubuntu/ {0}-security restricted multiverse universe main\n\
-         deb-src http://us.archive.ubuntu.com/ubuntu/ {0}-security restricted multiverse universe main\n\
-         deb http://us.archive.ubuntu.com/ubuntu/ {0}-backports restricted multiverse universe main\n\
-         deb-src http://us.archive.ubuntu.com/ubuntu/ {0}-backports restricted multiverse universe main\n\
-         deb http://apt.pop-os.org/proprietary {0} main\n",
-         release
-    ))?;
+pub fn create_new_sources_list(release: &str) -> io::Result<()> {
+    fs::write(
+        MAIN_SOURCES,
+        format!(
+            r#"# Ubuntu Repositories
+
+deb http://us.archive.ubuntu.com/ {0} restricted multiverse universe main
```

Comment:
I'm now getting `unable to upgrade to next release: failed to update sources` on every attempt, even though `sudo apt update` is not showing any errors. It looks like the four `us.archive.ubuntu.com` links need to have /ubuntu at the end like they used to.

Reply:
Fixed with https://github.com/pop-os/upgrade/pull/112/commits/2a8ffe53ef4e44b9ac25ed088eb9f223738b0392.